src/SFA.DAS.ApplyService.InternalApi/Services/AssessorLookupService.cs

```diff
@@ -0,0 +1,128 @@
+﻿using SFA.DAS.ApplyService.Application.Apply.Roatp;
+
+namespace SFA.DAS.ApplyService.InternalApi.Services
+{
+    public class AssessorLookupService : IAssessorLookupService
+    {
+        public string GetTitleForSequence(int sequenceId)
+        {
+            switch (sequenceId)
+            {
+                case RoatpWorkflowSequenceIds.ProtectingYourApprentices:
+                    return "Protecting your apprentices checks";
```

Comment:
I'd turn these into public strings, and return the public strings here (either as fields on this class, or probably better to create a separate class and reference it?)